sync/upstream.py

```diff
@@ -237,7 +231,7 @@ def get_or_create_remote_branch(self):
             while path in refs:
                 count += 1
                 path = "%s-%s" % (initial_path, count)
-            self.remote_branch = path[len("refs/remotes/origin/"):]
+            self.remote_branch = path[len("refs/remotes/origin/"):]  # type: ignore
```

Comment:
python 3.9 will have `removeprefix`

Reply:
We are still on 3.6 on the Py3 branch, upgrading more probably requires updating the base os image (I tried using deadsnakes, but ran into some issues that I'd prefer to resolve by updating the base).